pkg/external/external.go

```diff
@@ -5,10 +5,12 @@ import (
 	"encoding/json"
 	"fmt"
 	"strconv"
+
+	"github.com/aquasecurity/tracee/types"
 )
 
-// Event is a user facing data structure representing a single event
-type Event struct {
+// TraceeEvent is a user facing data structure representing a single event
+type TraceeEvent struct {
```

Comment:
Just to make sure I'm following, TraceeEvent is a struc that represent a payload of of event generated by tracee-ebpf?

Reply:
Yes. This is a name I agreed upon with @AsafEitani previously, however i'm open to suggestions for better names (I have my doubts about it after implementing this PR).

Reply:
I changed it back to `trace.Event` with @AsafEitani's refactor I thought the package name kinda fulfills the role instead of using `TraceeEvent` WDYT

Reply:
Resolved in #1493